packages/node_modules/@ciscospark/react-component-new-messages-separator/src/styles.css

```diff
@@ -1,8 +1,5 @@
-:root {
-  --color-primary: #049fd9;
-}
-
 .new-messages {
+  --color-primary: #049fd9;
   color: var(--color-primary);
```

Comment:
I don't think there was reasoning behind the variables honestly. Couldn't tell you my motivation especially since it isn't reused. I could just remove them.